### DISCUSSION TITLE
Quicklink on CLI output

### DISCUSSION THREAD

**Discussion by mvhirsch** [composer-unused/composer-unused#417] [Ideas]
I'm currently cleaning up multiple projects using composer-unused. I really love it :heart: 
Please keep on that good work!

`composer-unused` lists many packages I'm not sure if there is any indirect usage (like `twig/intl-extra` used in twig-templates automatically extended by Symfony Framework, ... or `egulias/email-validator` used in [`Constraint\Email(mode: strict)`](https://symfony.com/doc/5.4/reference/constraints/Email.html#mode), ... you get the idea).

![image](https://user-images.githubusercontent.com/573047/199198278-a17e7b3f-8e28-4f1c-94ca-91d6fb053005.png)

Like in `composer info icanhazstring/*`, there is a direct link to it's repository.
![image](https://user-images.githubusercontent.com/573047/199198182-778cd890-559f-41fe-82cb-aad1e84721a0.png)

It would be a great help, adding such links to the output of `composer-unused`, too!

**Comment by icanhazstring** [comment]
Hi. Thanks 🥳
Also this sounds like a nice idea.

Indirect usages, If per code basis, will be supported when the indirect usages scan is built in (#235) 😉

But there are some other occasions which are probably not that easy to scan for. For this, a direct link to those repos are actually quite nice to read the docs about usage. So that you can add them with more confidence to the "ignore" list of composer-unused.

**Reply by mvhirsch** [reply]
I'm not asking about indirect-usages-support (that's #235). Just asking about a little help in doing a manual research :grin:

**Reply by icanhazstring** [reply]
Sure. Was just mentioning again, as you mentioned indirect usage 😉

Ofc the link would be a quite nice addition.

**Comment by icanhazstring** [comment]
Already added links (if available)
